js/content/community.js

```diff
@@ -858,6 +878,58 @@ let ProfileHomePageClass = (function(){
     return ProfileHomePageClass;
 })();
 
+
+let GroupHomePageClass = (function(){
+
+    function GroupHomePageClass() {
+        this.addExtraLinks();
+    }
+
+    GroupHomePageClass.prototype.addExtraLinks = function() {
+
+        let groupId = GroupID.getGroupId();
+        let iconType = "none";
+        let images = SyncedStorage.get("show_profile_link_images");
+        if (images !== false) {
```

Comment:
There have been some updates on develop, we changed `false` to `"none"`
```suggestion
        if (images !== "none") {
```